(SSS) C:\Users\405B\KFC>python main.py --comment first_large_data --dataroot_dir ./data_large/
comment: first_large_data
model and loss plot -> ./model/model_first_large_data
Namespace(batch_size=100, beta1=0.5, beta2=0.999, comment='first_large_data', dataroot_dir='./data_large/', epoch=25, gpu_mode=True, lr=0.0002, num_workers=4, result_dir='./results/', sample_num=64, save_dir='./model/', type='train')
[*] Training started
[E000]  loss: 2.333589
[E001]  loss: 1.832898
[E002]  loss: 1.937395
[E003]  loss: 1.902936
[E004]  loss: 1.611142
[E005]  loss: 1.502762
[E006]  loss: 1.295950
[E007]  loss: 1.209875
[E008]  loss: 1.096231
[E009]  loss: 1.109720
[E010]  loss: 0.746361
[E011]  loss: 0.877750
[E012]  loss: 0.760421
[E013]  loss: 0.914819
[E014]  loss: 0.730388
[E015]  loss: 0.850092
[E016]  loss: 0.540594
[E017]  loss: 0.652777
[E018]  loss: 0.762748
[E019]  loss: 0.729174
[E020]  loss: 0.752393
[E021]  loss: 0.418854
[E022]  loss: 0.285025
[E023]  loss: 0.310931
[E024]  loss: 0.355296
Training finished!... save training results
Total time: 934.3001585006714
Per epoch time: [124.14700055122375, 38.116108417510986, 32.05409860610962, 32.32323932647705, 33.58890676498413, 32.90713334083557, 32.170440673828125, 31.152297973632812, 32.0348904132843, 32.8465473651886, 31.980900764465332, 34.09605860710144, 31.888433694839478, 32.352356910705566, 32.64029383659363, 33.249499559402466, 31.135193824768066, 31.76522469520569, 31.95205307006836, 31.930767059326172, 32.0590546131134, 32.93064594268799, 32.72220802307129, 31.171401977539062, 31.630552530288696]
[*] Training finished